.github/ISSUE_TEMPLATE/---mrtk-task.md

```diff
@@ -2,16 +2,27 @@
 name: "\U0001F4DD Mixed Reality Toolkit Task"
 about: File an outstanding task for MRTK vNext.
 ---
+> Necessary checks for creating a new vNext Task:
+> * Prefix the Title of the Task with "vNext Task: "
+> * Remove any defaulted test below, only include new information you are supplying for the task
```

Comment:
test -> text.

also, should likely be two sentences.

Reply:
Fixed